sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/configuration/StatsbeatConnectionString.java

```diff
@@ -88,16 +130,19 @@ public String getInstrumentationKey() {
     }
 
     // visible for testing
+    @Nullable
     static InstrumentationKeyEndpointPair getInstrumentationKeyAndEndpointPair(
         String customerEndpoint) {
         String geo = getGeoWithoutStampSpecific(customerEndpoint);
         if (geo != null && EU_REGION_GEO_SET.contains(geo.toLowerCase(Locale.ROOT))) {
             return new InstrumentationKeyEndpointPair(
                 EU_REGION_STATSBEAT_IKEY, EU_REGION_STATSBEAT_ENDPOINT);
+        } else if (geo != null && NON_EU_REGION_GEO_SET.contains(geo.toLowerCase(Locale.ROOT))) {
```

Comment:
If this is called often what I mentioned above may be better by default as this can use `String::compareToIgnoreCase` which doesn't have all the overhead `toLowerCase` does as comparing to ignoring case doesn't need to instantiate a new String

Reply:
this isn't called much so I think ok here